src/woo_publications/publications/api/serializers.py

```diff
@@ -421,6 +432,12 @@ class PublicationSerializer(serializers.ModelSerializer[Publication]):
             "otherwise an empty string is returned."
         ),
     )
+    kenmerken = PublicationIdentifierSerializer(
+        help_text=_("The publication identifiers attached to this publication."),
+        many=True,
+        source="publicationidentifier_set",
+        required=False,
+    )
```

Comment:
how many queries will we get now in the publicatie list endpoint? :-)

You'll want to update the viewset and add a `prefetch_related` to the queryset to not get wrecked by N + 1 queries.